### DISCUSSION TITLE
Publish websites on TOR only. Do nginx make sense ?

### DISCUSSION THREAD

**Discussion by BobSquarePants** [pallets/flask#4717] [Q&A] [answered]
Hi everyone,
I'm using Flask and Gunicorn. 
I would like to release my future creation on TOR only (to not be dependent of that centralized DNS sh*t)

I've read on https://flask.palletsprojects.com/en/2.1.x/deploying/

> WSGI servers have HTTP servers built-in. However, a dedicated HTTP server may be safer, more efficient, or more capable.
Putting an HTTP server in front of the WSGI server is called a “reverse proxy.”

Is still stand with TOR only approach ? as it's provide already a security layer..
Does using **nginx** will bring any benefits ?  Because it's goes against my moto : The less the better.

Flask - gunicorn - nginx

Thanks.

**Comment by davidism** [comment, marked as answer]
Whether you're accessing your app through Tor or any other network doesn't change that you should deploy it with production WSGI and HTTP servers.

**Reply by BobSquarePants** [reply]
Thank you @davidism !
